app/models/comment.rb

```diff
@@ -0,0 +1,9 @@
+class Comment < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+
+  def self.update_counter(post_id, value)
+    post = Post.where(id: post_id)
+    post.update(comments_counter: value)
+  end
```

Comment:
- [ ] The same happens with this too, consider updating it! 🤓 🔧 

- [ ] **[OPTIONAL]** It would be great to use the `after_save` callback here too! Try it 💯